sympy/core/power.py

```diff
@@ -135,7 +135,9 @@ class Pow(Expr):
     +--------------+---------+-----------------------------------------------+
     | 1**oo        | nan     | Because there are various cases where         |
     | 1**-oo       |         | lim(x(t),t)=1, lim(y(t),t)=oo (or -oo),       |
-    | 1**zoo       |         | but lim( x(t)**y(t), t) != 1.  See [3].       |
+    |              |         | but lim( x(t)**y(t), t) != 1.  See [3].       |
+    +--------------+---------+-----------------------------------------------+
+    | b**zoo       | nan     | b can be any general complex number.          |
```

Comment:
The third column should be the reason why its value is nan.

Reply:
The reason is given in the issue I raised: when b is not 1, b**z has no limit as |z|->oo when z is a complex number.  (And the case of b being 1 is already in the table).